### PR TITLE
Api 5360 add dropdown and revise color ratio

### PR DESCRIPTION
**Pull request by charleystran**
### Description
This PR adds the API dropdown selector to the top of the auth docs page. Additionally the PR changes the contrast ratio of the dropdown selector text and removed the rounded radius for the new API selector at the top of the page.

This PR related to https://vajira.max.gov/browse/API-5360

- [x] Tests added or updated for change
- [x] Docs updated
  - [x] If you have made structural changes to the site, the [Developer Portal Content Types](https://community.max.gov/x/qUwOg) docs have been updated
  - [x] If you have made/confirmed any decisions about user interactions and accessibility, the [Accessible Component Design](https://community.max.gov/x/xS8mgg) docs have been updated
- [x] Accessibility concerns have been considered and addressed
  
  - [x] If you've made any UI changes, you've run an axe check using the axe browser extension that reported no new issues

### CODE REVIEW

**Review by TS000 — APPROVED** [review]
All tests pass and the manual check looked great!

File: src/components/oauthDocs/BuildingOIDCContent.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ const BuildingOIDCContent = (): JSX.Element => {
   };
 
   return (
-    <section aria-labelledby="building-oidc-apps">
+    <section aria-labelledby="building-oidc-apps" id="building-oidc-apps">
```
Comment (mjlumetta) [inline src/components/oauthDocs/BuildingOIDCContent.tsx:23]: this id is also used by the section heading below. if we're using the section as the parent of all the code blocks, we need to choose a unique id here. we could also conceivably use a class, which is slightly preferable imo if we're using it only as a CSS selector (Scopes has a similar thing).

File: src/containers/documentation/AuthorizationDocs.scss [hunk 2]
```diff
@@ -58,15 +58,17 @@
...
+  }
+  .api-selector {
+    background-color: $color-gray;
 
     select {
-      color: $color-white;
+      color: #80f8a2;
       option {
-          color: $color-base;
+          color: #80f8a2;
```
Comment (mjlumetta) [inline src/containers/documentation/AuthorizationDocs.scss:71]: this should remain `$color-base`, i believe. depends on your browser, but the option color can be what shows up when you open the select options. mine looks fine on Chrome, but here's what it looks like in Firefox:

Screen Shot 2021-02-26 at 9 42 19 AM

cc @ediiotero if you have any other explanation or context to add on the styling patterns here

**Review by mjlumetta — CHANGES_REQUESTED** [review]
couple of small but important changes to suggest here. lmk if you have thoughts on those considerations

File: src/containers/documentation/AuthorizationDocs.scss (hunk 2 above)
Comment (charleystran) [reply src/containers/documentation/AuthorizationDocs.scss:71]: This has been been fixed

File: src/components/oauthDocs/BuildingOIDCContent.tsx (hunk 1 above)
Comment (charleystran) [reply src/components/oauthDocs/BuildingOIDCContent.tsx:23]: I changed to use a class instead.

**Review by mjlumetta — APPROVED** [review]

**Review by gileswells — APPROVED** [review]
LGTM

**Pull request merged**